modules/selenium/src/test/java/org/testcontainers/junit/BaseWebDriverContainerTest.java

```diff
@@ -49,12 +47,8 @@ protected static void doSimpleExplore(BrowserWebDriverContainer<?> rule, Capabil
         driver.quit();
     }
 
-    protected void assertBrowserNameIs(BrowserWebDriverContainer<?> rule, String expectedName) {
-        Capabilities capabilities = ("chrome".equals(expectedName)) ? new ChromeOptions() : new FirefoxOptions();
-        RemoteWebDriver driver = new RemoteWebDriver(rule.getSeleniumAddress(), capabilities);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(30));
-        String actual = driver.getCapabilities().getBrowserName();
+    protected void assertBrowserNameIs(String expectedName, Capabilities capabilities) {
+        String actual = capabilities.getBrowserName();
```

Comment:
previous check was against the driver instead of directly to the capabilities. I think the driver will check against the browser running in the container.

Reply:
Yep, I totally misunderstood the concept of the test, also that webDriver and caps return a different name for edge confused me a little bit